static/templates/settings/bot-settings.handlebars

```diff
@@ -25,6 +25,13 @@
                 <h4 class="new-bot-section-title light no-margin">{{t "Add a new bot" }}</h4>
                 <div class="new-bot-form">
                     <div class="input-group">
+                        <label for="bot_type">{{t "Bot type" }}</label>
+                        <select name="bot_type" id="create_bot_type">
+                            <option value="1">{{t "Custom bot" }}</option>
+                            <option value="2">{{t "Incoming webhook bot" }}</option>
```

Comment:
I changed the second one to "Generic bot" and "Incoming webhook", for clarity.  I also added a tooltip that explains what the difference is.